packages/babel-helper-compilation-targets/src/index.ts

```diff
@@ -169,6 +170,18 @@ function resolveTargets(queries: Browsers, env?: string): Targets {
   return getLowestVersions(resolved);
 }
 
+const targetsCache = new LruCache(64);
```

Comment:
We probably need less than 64, since the targets are usually the same for every file (I believe that 8 would be enough). However it doesn't really matter, since 64 small allocations already has a very small memory footprint.